app/src/main/res/values-fr/strings.xml

```diff
@@ -640,4 +640,24 @@ Ces formulaires avant de télécharger une version plus récente. </string>
     <string name="warning_switch_server">Changement de serveur: Cela effacera toutes les données mUzima sur votre appareil, y compris les formulaires incomplets. Etes-vous sûr de vouloir modifier ce paramètre?</string>
     <string name="write_SHR_to_card_text">Ecrire SHR à la carte</string>
     <string name="write_shr_to_card_general">Ecrire des dossiers de santé partagés à la carte</string>
+    <string name="feedback_header">S\'il vous plaît envoyez-nous vos commentaires et réactions</string>
+    <string name="feedback_rating_question1">Comment facile à utiliser mUzima App?</string>
```

Comment:
Hi Zita, rename feedback_rating_question1 to feedback_rating_question

Reply:
Hello Benard, I have changed it.

Reply:
Hi @rezita, Rename in the other strings.xml

Reply:
Hello @benamonya, I did it.